curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.md

```diff
@@ -12,7 +12,7 @@ ES6 provides a new syntax to create objects, using the <dfn>class</dfn> keyword.
 
 It should be noted that the `class` syntax is just syntax, and not a full-fledged class-based implementation of an object-oriented paradigm, unlike in languages such as Java, Python, Ruby, etc.
 
-In ES5, we usually define a `constructor` function and use the `new` keyword to instantiate an object.
+In ES5, an object can be created by defining a `constructor` function and using the `new` keyword to instantiate the object.
```

Comment:
The old ES5 example can be deleted, to simplify this and reduce the text

Reply:
I think we should keep it for reference.
If you still want to remove it, let me know I'll do it.

Reply:
I still want it removed, as discussed in the issue. 

It is best to follow the decisions from the issue when you make a contribution.

Reply:
removed